python/tvm/script/parser.py

```diff
@@ -536,7 +536,7 @@ def transform_SubscriptAssign(self, node):
             if len(indexes) != 1:
                 self.report_error(
                     f"Store is only allowed with one index, but {len(indexes)} were provided.",
-                    tvm.ir.Span.union([x.span for x in indexes]),
+                    node.params[1].span,
```

Comment:
```suggestion
                    ast.Span.union([x.span for x in node.params]),
```

You want the union of all the spans, not just the first one.